firmware/README.md

```diff
@@ -389,6 +390,21 @@ The IM21 door sensors each have a sticker on them with their door IDs. On the bo
 - 1 - if door ID echoed to the cloud
 - -1 - if bad input was received and door ID was neither parsed or echoed to the cloud
 
+### **force_reset(String)**
```

Comment:
Thanks for noticing this and updating the docs!

Reply:
No worries